src/main/java/io/appium/java_client/remote/MobileCapabilityType.java

```diff
@@ -124,7 +124,7 @@
     String EVENT_TIMINGS = "eventTimings";
```

Comment:
Please revert this file. It has been merged in the other PR

Reply:
Sure, but git magics it away when/if you squash/rebase. :)